.circleci/config.yml

```diff
@@ -286,8 +286,8 @@ workflows:
           ruby_opt: -W:deprecated
       - tests_ubuntu:
           name: 'Execute tests on Ubuntu'
-          image: 'fastlanetools/ci:0.3.0'
-          ruby_version: '2.6'
+          image: 'circleci/ruby:2.7.5-node'
```

Comment:
No need in custom docker container. `fastlanetools/ci` is locked on Ruby 2.6 and failing with v2.7.